docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -32,6 +32,8 @@ Use the name of the container for the start, stop, and description in the unit f
 
 Using this flag will yield unit files that do not expect containers and pods to exist.  Instead, new containers and pods are created based on their configuration files.  The unit files are created best effort and may need to be further edited; please review the generated files carefully before using them in production.
 
+Note that `--new` only works on containers and pods created directly via Podman.  It does not work on container or pods created via the REST API.
```

Comment:
Should we also mention it does not work with `play kube`?

Reply:
Ooooh yes we should :)  Thanks!

Reply:
Updated